packages/cli/src/platform/auth/show.ts

```diff
@@ -13,10 +14,20 @@ export class Show implements Command {
     })
     if (isError(args)) return args
     const token = await getPlatformTokenOrThrow(args)
-    return platformRequestOrThrow({
+    const payload = await platformRequestOrThrow<{
+      user: { id: string; handle: string; email: string; displayName: string }
+    }>({
       token,
       path: `/settings/account`,
       route: '_app._user.settings.account',
-    }) as Promise<any>
+    })
+    console.log(`Currently authenticated as ${green(payload.user.email)}`)
+    console.log('')
```

Comment:
```suggestion
    console.info(`Currently authenticated as ${green(payload.user.email)}\n`)
```
Use `console.info` when you want to log something to the user. It makes it easier to spot random `console.log` from debugging for example.